airflow/api_connexion/endpoints/forward_to_fab_endpoint.py

```diff
@@ -0,0 +1,124 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Could you add tests for this file?

Reply:
I thought about it, but decided not to, because
 - it contains no logic, the test would just be repeating what's in the file (checking that we're calling the right method I suppose ?), adding no value.
 - it's just some deprecation scaffolding, not meant to persist in time and be maintained forever.

But if you think those are not good enough arguments, I can add a test file, sure.

Reply:
Nop, I can buy that :)